extension/src/popup/views/SignTransaction.tsx

```diff
@@ -86,7 +93,13 @@ export const SignTransaction = () => {
   const location = useLocation();
   const dispatch = useDispatch();
   const { transaction, domain } = getTransactionInfo(location.search);
+
   const punycodedDomain = punycode.toASCII(domain);
+  const whitelistStr = localStorage.getItem(WHITELIST_ID) || "";
+  const whitelist = whitelistStr.split(",");
+  const isDomainWhiteListed = whitelist.includes(
+    removeQueryParam(punycodedDomain),
+  );
```

Comment:
same - we can merge for right now, but we'll need to take this out in a followup ticket

Reply:
Updated docs and dependency for `lyra-api`